tests/pkg_tiny-asn1/main.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (C) 2016 Mathias Tausig, FH Campus Wien
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v3. See the file LICENSE in the top level
```

Comment:
I changed the licence to LGPL v2.1. The testcase probably won't ever be the
subject of any legal battle ;-)

On Don, 2016-09-01 at 04:54 -0700, Martine Lenders wrote:
> > 
> > @@ -0,0 +1,208 @@
> > +/*
> > + * Copyright (C) 2016 Mathias Tausig, FH Campus Wien
> > + *
> > + * This file is subject to the terms and conditions of the GNU Lesser
> > + * General Public License v3. See the file LICENSE in the top level
> Sorry... for some reason Murdock only runs the static tests one by one
> ...  The rest of RIOT is  licensed under LGPLv2.1. Would you mind changing
> this file to that license too? Otherwise you have to include the full license
> text for LPGLv3 + add a pattern to dist/tools/licenses/patterns. But IANAL and
> it might be, that there are some good reasons, why we went for LGPLv2.1 and
> there might be incompatibilities to LGPLv3, so changing this files licensing
> might be more desirable.
>